packages/jest-expo-enzyme/src/setupEnzyme.web.ts

```diff
@@ -2,6 +2,8 @@ import Adapter from 'enzyme-adapter-react-16';
 import Enzyme from 'enzyme';
 import serializer from './serializer';
 
+declare const global: { expect: any };
+  
 Enzyme.configure({ adapter: new Adapter() });
 
-expect.addSnapshotSerializer(serializer);
+global.expect.addSnapshotSerializer(serializer);
```

Comment:
revert, `expect` alone should work with `@types/jest` but may need some config overrides since we add `@types/jest` only in `__tests__` by default

Reply:
I don't know if this works like you think it does 🤔